docs/examples/git-repo/README.md

```diff
@@ -0,0 +1,84 @@
+## Usage 
+This example is modified on the basis of kustomized controller provided by flux.
+The kustomize-controller is part of a composable GitOps toolkit and depends on source-controller 
+to acquire the Kubernetes manifests from Git repositories and S3 compatible storage buckets.
+
+### 1. Install toolkit controllers
+Download the flux CLI:
+```shell
+curl -s https://fluxcd.io/install.sh | sudo bash
+```
+Install the toolkit controllers in the flux-system namespace
+```shell
+flux install
+```
+
+### 2. Define a Git repository source
+Create a source object that points to a Git repository containing Kubernetes and Kustomize manifests
+Cause we want to verify the gitops logic through git commit && git push to our own repository,
+we need to fork the official warehouse to our own warehouse.
+```shell
+fork  https://github.com/stefanprodan/podinfo  to your own repository
+git clone https://github.com/$(yourgithubname)/podinfo
+---
+# edit in  ./gitrepo-kind.yaml and 
+# change "url: https://github.com/stefanprodan/podinfo" with "url: https://github.com/$(yourgithubname)/podinfo"
+---
+kubectl apply -f gitrepo-kind.yaml
+```
+You can wait for the source controller to assemble an artifact from the head of the repo master branch with
+```shell
+kubectl -n flux-system wait gitrepository/podinfo --for=condition=ready
+```
+The source controller will check for new commits in the master branch every minute. You can force a git sync with
```

Comment:
Please write a complete sentence for the second part of this line.

Reply:
the line hopes to express "you can force a git sync with the instruction below" I should add a ":" after "with".